packages/block-editor/src/components/use-block-drop-zone/index.js

```diff
@@ -456,7 +456,15 @@ export default function useBlockDropZone( {
 				const [ targetIndex, operation, nearestSide ] =
 					dropTargetPosition;
 
-				if ( isZoomOut() && operation !== 'insert' ) {
+				const isTargetIndexEmptyDefaultBlock =
+					operation === 'replace' &&
+					blocksData[ targetIndex ]?.isUnmodifiedDefaultBlock;
```

Comment:
It's a bit confusing to read operation === 'replace' and later operation !== 'insert' and also why does the kind of operation determine if isTargetIndexEmptyDefaultBlock, as it feels the isUnmodifiedDefaultBlock flag should be enough.

Reply:
You know what I think we might be able to remove the `operation` check like you suggest.

Reply:
✅ Does it work for you?